components/image/graphviz/graphviz-python-27.p5m

```diff
@@ -43,8 +43,13 @@ file path=usr/lib/graphviz/python27/gv.py
 file path=usr/lib/graphviz/python27/libgv_python27.so
 
 file usr/lib/python2.7/vendor-packages-32/_gv.so path=usr/lib/python2.7/vendor-packages/_gv.so
-file usr/lib/python2.7/vendor-packages-32/gv.py path=usr/lib/python2.7/vendor-packages/gv.py pkg.tmp.autopyc=false
+file usr/lib/python2.7/vendor-packages-32/gv.py  path=usr/lib/python2.7/vendor-packages/gv.py     pkg.tmp.autopyc=false
 file usr/lib/python2.7/vendor-packages-64/_gv.so path=usr/lib/python2.7/vendor-packages/64/_gv.so pkg.linted.userland.action001.2=true
+file usr/lib/python2.7/vendor-packages-64/gv.py  path=usr/lib/python2.7/vendor-packages/64/gv.py  pkg.tmp.autopyc=false
```

Comment:
no need to deliver usr/lib/python2.7/vendor-packages/64/gv.py, 64-bit python  will use usr/lib/python2.7/vendor-packages/gv.py